config/development.json

```diff
@@ -102,7 +101,6 @@
 		"perfmon": false,
 		"persist-redux": true,
 		"plans/personal-plan": true,
-		"plans/wordpress-ad-credits": true,
```

Comment:
Sorry, it's related because it's another part of the "advertising vouchers" feature that never launched. I'm removing it in this PR as well, as it uses much of the same plumbing.